src/Components/build/repo.props

```diff
@@ -0,0 +1,55 @@
+﻿<Project>
+  <Import Project="dependencies.props" />
+
+  <!-- Properties to make KoreBuild more like dotnet/arcade. -->
+  <PropertyGroup>
+    <BuildDir>$(ArtifactsDir)$(Configuration)\packages\</BuildDir>
```

Comment:
```suggestion
```
Aha, here's the other one that should be removed.

Reply:
That wasn't enough on its own, but tracked it down to needing to add `<OverridePackageOutputPath>false</OverridePackageOutputPath>`. Hopefully will work now - waiting to see.